src/main/external-resources/renderers/Cling.conf

```diff
@@ -0,0 +1,38 @@
+#----------------------------------------------------------------------------
```

Comment:
Should this be removed before merge?

Reply:
@SubJunk no problem but it is in the each renderer confs.

Reply:
Sorry I meant the file itself. Is it meant to be merged?